src/ucs/debug/memtrack.c

```diff
@@ -255,25 +256,34 @@ void ucs_memtrack_releasing(void* ptr)
     pthread_mutex_unlock(&ucs_memtrack_context.lock);
 }
 
+static UCS_F_ALWAYS_INLINE void ucs_memtrack_releasing_int(void *ptr)
+{
+    if ((ptr == NULL) || !ucs_memtrack_is_enabled()) {
```

Comment:
maybe check for ptr==NULL inside do_{allocated|releasing} functions ?

Reply:
> maybe check for ptr==NULL inside do_{allocated|releasing} functions ?

done